builds/package-download/download-packages.proj

```diff
@@ -7,13 +7,8 @@
     <!-- Download any runtime packs as specified using the PackageRuntimeIdentifiers property -->
     <PackageRuntimeIdentifiers Include="$(PackageRuntimeIdentifiers.Split(' '))" />
 
-    <!-- Download the mono osx pack as well -->
-    <!-- I wasn't able to figure out how to only do this if 'osx-x64'/'osx-arm64' is already included -->
-    <PackageRuntimeIdentifiers Include="Mono.osx-x64" />
-    <PackageRuntimeIdentifiers Include="Mono.osx-arm64" />
-
     <!-- download the runtime packs -->
-    <PackageDownload Include="@(PackageRuntimeIdentifiers -> 'Microsoft.NETCore.App.Runtime.%(Identity)')" Version="[$(BundledNETCorePlatformsPackageVersion)]" />
+    <PackageDownload Include="@(PackageRuntimeIdentifiers -> 'Microsoft.NETCore.App.Runtime.Mono.%(Identity)')" Version="[$(BundledNETCorePlatformsPackageVersion)]" />
```

Comment:
It looks like this doesn't download the CoreCLR version anymore (which we need for macOS). However, that should make the build fail, which it apparently doesn't, so IDK?

Reply:
were any test are enabled ? maybe it silently fail ? 🤷‍♂️